### DISCUSSION TITLE
Contra layout?

### DISCUSSION THREAD

**Discussion by apiep** [manna-harbour/miryoku#45] [Q&A] [answered]
Is there a way to generate one for contra? It has the same layout as planck, so it should not be difficult to generate one for contra.

## Feature Request Type

- [ ] Core functionality
- [ ] Add-on hardware support (eg. audio, RGB, OLED screen, etc.)
- [x] Alteration (enhancement/optimization) of existing feature(s)
- [ ] New behavior

**Comment by manna-harbour** [comment, marked as answer]
Thanks for your request.  

The contra should already be covered by the [ortho_4x12 layout](https://github.com/manna-harbour/qmk_firmware/blob/miryoku/users/manna-harbour_miryoku/miryoku.org#ortho_4x12).  Build with `make contra:manna-harbour_miryoku`.

Let me know if there's any issues.

**Comment by apiep** [comment]
Ahh	yes, that works. I didn't know i could compile qmk with `[kb]:[user]` thanks

**Comment by manna-harbour** [comment]
You're welcome.  You can build with all the directories `keyboards/*/keymaps/*/` and, as in this case, `layouts/community/*/*/`.  

Also, from your qmk fork it looks like you had an iris as well.  I just added [iris support](https://github.com/manna-harbour/qmk_firmware/blob/miryoku/users/manna-harbour_miryoku/miryoku.org#iris) if you'd like to try that too.

**Comment by apiep** [comment]
Yes, I will try that. But not for now, atleast my iris in a ready state. It is currently in a state when you wanna desolder but still did not have the mood to continue. 😅
